app/controllers/concerns/waste_carriers_engine/token_functionality.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module WasteCarriersEngine
+  module TokenFunctionality
```

Comment:
Can this be renamed to align with the usual concern naming style, starting with `CanHave` ?